src/main/scala/io/picnicml/doddlemodel/data/CsvLoader.scala

```diff
@@ -51,6 +51,13 @@ object CsvLoader {
     FeatureIndex(featureNames, featureTypes, featureNames.indices.toList)
   }
 
+  /** Transforms Feature Values with non-numerical type (Eg Categorical types) to a numerical value
```

Comment:
Maybe we should avoid using the word `transforms` here as this function actually creates the `LabelEncoder` which is then used to do the transformation that you described? I believe something like `creates a label encoder for a given dataset...` would better describe the functionality.